src/workerd/jsg/jsg.h

```diff
@@ -48,6 +48,35 @@ kj::String KJ_STRINGIFY(v8::Local<T> value) {
 
 namespace workerd::jsg {
 
+// The following APIs are available to be used for testing purposes for
+// v8 fast API. Please take a look into v8-fast-api.h file in V8 for more
+// information.
+namespace fast_api {
+
+namespace {
```

Comment:
Anonymous namespaces do not work in header files. This will cause the content of the namespace to be cloned into every translation unit, creating code bloat.

Reply:
Any recommendation on where to move this, or are you ok with the current place?

Reply:
I agree it shouldn't be in jsg.h, since jsg.h is supposed to be the public interface. I don't have a specific alternative suggestion.